server/controllers/finance/receipts/list.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
This file is actually a report of all invoices, correct?   I would put it in `/finance/reports/invoices` to avoid developer confusion.